baker/formatWordpressPost.tsx

```diff
@@ -476,6 +477,24 @@ export const formatWordpressPost = async (
         }
     })
 
+    const expandableParagraphs = cheerioEl('block[type="expandable-paragraph"]')
+    expandableParagraphs.each((_, eP) => {
+        const $el = cheerioEl(eP)
+        const $dry = cheerioEl(
+            ReactDOMServer.renderToStaticMarkup(
+                <div>
+                    <ExpandableParagraph
+                        dangerouslySetInnerHTML={{
+                            __html: $el.html() || "",
+                        }}
+                    />
+                </div>
+            )
+        )
+        $el.after($dry)
+        $el.remove()
+    })
```

Comment:
I generally abstract this type of processing into renderer functions (see https://github.com/owid/owid-grapher/blob/1532-expandable/baker/formatWordpressPost.tsx#L306) for conciseness.